### ISSUE TITLE
displaying issues with Python 3.9

### ISSUE THREAD

**Issue by leon-thomm** [UN-GCPDS/qt-material#17]
Hey! Two things.

1. On both python versions, the scale of QRadioButtons as well as QCheckBoxes is larger when using PySide6 compared to PySide2 (it looks like with PySide2 it's missing some styles).

PySide6
![grafik](https://user-images.githubusercontent.com/29960068/107352925-b8216100-6acc-11eb-95d0-3bda2a34c1e9.png)

PySide2
![grafik](https://user-images.githubusercontent.com/29960068/107352896-b061bc80-6acc-11eb-9a36-882443dc87a3.png)

2. When using Python 3.9 it doesn't draw the boxes at all, neither with PySide2, nor with PySide6
![grafik](https://user-images.githubusercontent.com/29960068/107353439-54e3fe80-6acd-11eb-93aa-608986a71759.png)

Code:
```
from PySide2.QtWidgets import QApplication, QMainWindow, QRadioButton, QWidget, QVBoxLayout, QCheckBox
import sys
import qt_material


class MainWindow(QMainWindow, qt_material.QtStyleTools):
    def __init__(self):
        QMainWindow.__init__(self)
        qt_material.QtStyleTools.__init__(self)

        widget = QWidget()
        widget.setLayout(QVBoxLayout())
        widget.layout().addWidget(QRadioButton('radio button 1'))
        widget.layout().addWidget(QRadioButton('radio button 2'))
        cbc = QCheckBox('check box 1, checked')
        cbc.setChecked(True)
        widget.layout().addWidget(cbc)
        widget.layout().addWidget(QCheckBox('check box 1, unchecked'))
        self.setCentralWidget(widget)


if __name__ == '__main__':
    app = QApplication()
    qt_material.apply_stylesheet(app, theme='dark_blue.xml')
    mw = MainWindow()
    mw.show()

    sys.exit(app.exec_())
```
for this I installed qt-material globally for both python versions using pip

**Comment by YeisonCardona** [comment]
Hi @leon-thomm 

1. There is no reason for the differences between PySide2 and PySide 6, can you please confirm that both windows look exactly the same without any theme?, maybe the `app.setStyle('Fusion')` in one of them is the problem, I just remove this feature recently.

2. Please check the installation of `qt5-svg` and `qt6-svg` and verify that from Python 3.9 and PySideX you can handler other SVG images.

I'm so sorry that I can't make this test by myself, I don't have a windows machine.

**Comment by leon-thomm** [comment]
1. Oh, yes you are right, sorry, it's a PySide thing, seems like they changed some default styling for PySide6. my bad.
2. I'm not sure how to check that? The QtSvg module is there, and I can create and display SvgWidgets (in PySide2, I think they removed it in PySide6, it's not there anymore). Anything specific I should check?

**Comment by leon-thomm** [comment]
apparently my python installation was a bit bugged. PySide2 (and PySide6) failed to load some (existing) svg modules on Python 3.9, which was resolved by a clean reinstall of Python 3.9 and the packages
